docs/template_plugin/tests/functional/op_reference/conversion.hpp

```diff
@@ -17,15 +17,24 @@ namespace reference_tests {
 namespace ConversionOpsRefTestDefinitions {
 
 static std::map<ngraph::helpers::ConversionTypes, std::string> conversionNames = {
-    {ngraph::helpers::ConversionTypes::CONVERT,      "Convert"},
-    {ngraph::helpers::ConversionTypes::CONVERT_LIKE, "ConvertLike"}
-};
```

Comment:
Seems we don't have style rules for that.
I'm not sure which one is recommended now, but anyway this small custom style changes in this PR makes it harder to review. A lot of changed lines in `op_reference/conversion.hpp` but the actual change is one letter in the line number 50 `convertionType` -> `conversionType`  :shipit:

Reply:
You're right - I have removed unnecessary changes and disabled auto-clang-format on save so it won't happen again.